cmd/pulumi-test-language/testdata/l2-failed-create-continue-on-error/main.pp

```diff
@@ -0,0 +1,39 @@
+resource "failing" "fail_on_create:index:Resource" {
+    value = true
+}
+
+resource "dependent" "simple:index:Resource" {
+    value = true
+    options {
+        dependsOn = [failing]
+    }
+}
+
+resource "dependent_on_output" "simple:index:Resource" {
+    value = failing.value
+}
+
+resource "independent" "simple:index:Resource" {
+    value = true
+}
+
+resource "independent2" "simple:index:Resource" {
+    value = true
+    options {
+        dependsOn = [independent]
+    }
+}
+
+resource "independent3" "simple:index:Resource" {
+    value = true
+    options {
+        dependsOn = [independent2]
+    }
+}
```

Comment:
Feel like this is probably overkill? If `independent` works how would these not work?

Reply:
It's the other way round.  At least in my testing `up` without `--continue-on-error` would sometimes be able to create `independent` and even `independent2`, even when `failing` failed earlier.  So I added these mostly for the purpose of showing that this works now, but doesn't work without `--continue-on-error`.  

We do have engine tests for that as well, so this might be slightly overkill here, but a couple more resources shouldn't make the test too much slower I think, so it felt better to just keep them.

Reply:
That's testing race semantics I think, resource registration from the SDKs don't have a defined order unless explicit dependencies are present.